graphql_compiler/query_formatting/common.py

```diff
@@ -32,39 +32,49 @@ def validate_argument_type(expected_type, value):
     Args:
         expected_type: GraphQLType we expect. All GraphQLNonNull type wrappers are stripped.
         value: object that can be interpreted as being of that type
+        name: optional string. If it is a string, it is the name of the argument and will be used to
```

Comment:
I decided to make this parameter optional in order to order to make this change backwards compatible since any non-backward compatible change to a public api function requires a major version bump, (and according to lines 20-22 this function is part of the public API).